dias/analyzers/feedpositions_analyzer.py

```diff
@@ -210,6 +214,13 @@ def run(self):
                     date=self.start_time_night, deg=True)
             if transit:
                 night_transits.append(src)
+            # At this point it's also good to set all metrics for all sources
```

Comment:
I am not sure if it's a good idea to export `0`s for all metrics every time. If this is to make sure all metrics are `0` in the case of #109 , the better solution might be setting them to `0` there.

Reply:
replacing it with a function called __set_metrics_to_zero and use it whenever necessary.